img/a/au.com.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generator: Adobe Illustrator 25.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="レイヤー_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 60 22" style="enable-background:new 0 0 60 22;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#EB5505;}
+</style>
+<path class="st0" d="M60,7.3c-0.2,0.4-0.4,0.6-0.6,0.9c-2.2,3.2-4.5,6.4-7.1,9.3c-1.1,1.2-2.3,2.4-3.8,3.2c-1.5,0.8-3.2,1.1-4.9,0.8  c-1.8-0.3-3-1.3-3.4-3.1c0-0.1-0.1-0.2-0.1-0.4c-0.1,0.1-0.3,0.2-0.4,0.4c-2.8,3-6.3,4-10.3,3.1c-1.8-0.4-2.9-1.5-3.4-3.3l-0.1-0.1  c-0.1,0.2-0.2,0.3-0.3,0.5c-1.7,1.9-3.8,3.1-6.4,3.2c-2.5,0.1-4.1-1-4.9-3.6c-0.1,0.1-0.2,0.2-0.3,0.3c-2.5,2.5-5.5,3.6-9,3.3  c-0.8-0.1-1.6-0.3-2.3-0.6c-1.8-0.8-2.8-2.5-2.7-4.5C0.5,9.4,5.6,3.2,12.7,1.3c2.3-0.6,4.7-0.9,7.2-1c1.4,0,2.8,0,4.2,0  c-0.2,0.9-0.5,1.9-0.7,2.9c-1.1,4.3-2.3,8.7-3.4,13c-0.1,0.6-0.2,1.2-0.2,1.7c0,0.9,0.5,1.3,1.4,0.9c1.2-0.5,2.1-1.3,3-2.2  c1.1-1.2,1.7-2.6,2.1-4.2c0.6-2.3,1.1-4.5,1.8-6.7c0.5-1.8,1.5-3.3,3.1-4.3c1.4-0.9,2.9-1.2,4.6-1.1c-0.2,0.9-0.4,1.7-0.6,2.5  c-1.1,4.3-2.3,8.7-3.4,13c-0.1,0.5-0.2,1.1-0.2,1.7c0,1.1,0.6,1.6,1.7,1.6c1,0,1.8-0.4,2.7-0.8c2.4-1.3,3.8-3.4,4.6-5.9  c0.6-1.8,1-3.7,1.5-5.5C42.5,5.4,43,4,44,2.8c1.6-1.9,3.6-2.6,6.1-2.5c-0.3,1-0.5,2-0.8,3c-1.1,4.3-2.3,8.7-3.4,13  c-0.1,0.5-0.2,1.1-0.2,1.7c0,0.8,0.5,1.1,1.2,0.9c1.1-0.3,1.9-1,2.6-1.7c1.8-1.7,3.3-3.8,4.8-5.8c0.9-1.2,1.7-2.4,2.5-3.7  c0.1-0.2,0.3-0.3,0.5-0.3C58.1,7.3,59,7.3,60,7.3z M17.1,2.7c-0.4,0.1-0.8,0.1-1,0.2c-2.3,0.8-4.4,2.1-6,4c-2,2.3-3.1,4.9-3.9,7.8  c-0.2,0.9-0.4,1.8-0.5,2.8c-0.1,1.2,0.7,1.8,1.9,1.7c1.2-0.1,2.2-0.6,3.2-1.3c2.1-1.5,3.3-3.6,4-6c0.7-2.6,1.4-5.2,2.1-7.8  C16.9,3.7,17,3.3,17.1,2.7z"/>
+</svg>
```

Comment:
[SVGOMG](https://jakearchibald.github.io/svgomg/) is a great tool for minifying SVG files before uploading them here.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 22"><path fill="#eb5505" d="m60 7.3-.6.9a91.4 91.4 0 0 1-7.1 9.3 14.7 14.7 0 0 1-3.8 3.2c-1.5.8-3.2 1.1-4.9.8-1.8-.3-3-1.3-3.4-3.1l-.1-.4-.4.4c-2.8 3-6.3 4-10.3 3.1a4.3 4.3 0 0 1-3.4-3.3l-.1-.1-.3.5a8.9 8.9 0 0 1-6.4 3.2c-2.5.1-4.1-1-4.9-3.6l-.3.3a10.9 10.9 0 0 1-9 3.3c-.8-.1-1.6-.3-2.3-.6A4.5 4.5 0 0 1 0 16.7C.5 9.4 5.6 3.2 12.7 1.3c2.3-.6 4.7-.9 7.2-1h4.2l-.7 2.9-3.4 13-.2 1.7c0 .9.5 1.3 1.4.9a9.3 9.3 0 0 0 3-2.2 9.3 9.3 0 0 0 2.1-4.2c.6-2.3 1.1-4.5 1.8-6.7A7.1 7.1 0 0 1 35.8.3l-.6 2.5-3.4 13-.2 1.7c0 1.1.6 1.6 1.7 1.6 1 0 1.8-.4 2.7-.8a9.7 9.7 0 0 0 4.6-5.9c.6-1.8 1-3.7 1.5-5.5.4-1.5.9-2.9 1.9-4.1A7 7 0 0 1 50.1.3l-.8 3-3.4 13-.2 1.7c0 .8.5 1.1 1.2.9a6 6 0 0 0 2.6-1.7c1.8-1.7 3.3-3.8 4.8-5.8l2.5-3.7c.1-.2.3-.3.5-.3l2.7-.1zM17.1 2.7l-1 .2c-2.3.8-4.4 2.1-6 4-2 2.3-3.1 4.9-3.9 7.8l-.5 2.8c-.1 1.2.7 1.8 1.9 1.7 1.2-.1 2.2-.6 3.2-1.3 2.1-1.5 3.3-3.6 4-6l2.1-7.8.2-1.4z"/></svg>
```